### PR TITLE
Edit language

### PR DESCRIPTION
**Pull request by ghost**
## Description
Edit language
## Reference Requirements
- Product Backlog ID: Non-PBI
- Task ID: Non-TI

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- update `languages` set `language` = 'Englishd', `updated_at` = '2018-03-07 08:29:20' where `id` = '1'
image

### CODE REVIEW

**Comment by at-caonguyen** [comment]
LGTM :+1:

File: app/Http/Controllers/Admin/LanguageController.php [hunk 1]
```diff
@@ -26,6 +28,27 @@ public function index()
...
+     * Update infomation of Language.
+     *
+     * @param App\Http\Requests\Backend\EditLanguageRequest $request  request
+     * @param App\Model\Language                            $language language
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function update(EditLanguageRequest $request, Language $language)
+    {
+        DB::beginTransaction();
```
Comment (at-vietpham) [inline app/Http/Controllers/Admin/LanguageController.php]: Tại sao đoạn này lại cần dùng `transaction`?

File: app/Http/Controllers/Admin/LanguageController.php [hunk 2]
```diff
@@ -26,6 +28,27 @@ public function index()
...
+        $result = false;
+        try {
+            $result = $language->update($request->only('language'));
+            DB::commit();
+        } catch (Exception $e) {
+            DB::rollBack();
```
Comment (at-vietpham) [inline app/Http/Controllers/Admin/LanguageController.php]: Rollback xong không show ra message gì hết à.

**Comment by at-vietpham** [comment]
LGTM 🏷

**Pull request merged**